contracts/core/ManualVolOracle.sol

```diff
@@ -31,44 +45,63 @@ contract ManualVolOracle is AccessControl {
      * @notice Returns the standard deviation of the base currency in 10**8 i.e. 1*10**8 = 100%
      * @return standardDeviation is the standard deviation of the asset
      */
-    function vol(address) public pure returns (uint256 standardDeviation) {
+    function vol(bytes32) public pure returns (uint256 standardDeviation) {
         return 0;
     }
 
     /**
      * @notice Returns the annualized standard deviation of the base currency in 10**8 i.e. 1*10**8 = 100%
      * @return annualStdev is the annualized standard deviation of the asset
      */
-    function annualizedVol(address pool)
+    function annualizedVol(bytes32 instrumentId)
         public
         view
         returns (uint256 annualStdev)
     {
-        return annualizedVols[pool];
+        return annualizedVols[instrumentId];
+    }
+
+    /**
+     * @notice Computes the instrument id for a given Instrument struct
+     * @param instrument is an Instrument struct to encode
+     */
+    function getInstrumentId(Instrument calldata instrument)
```

Comment:
Maybe add a duplicate function that let's us pass in the `Instrument` so we don't have to abi encode the calldata for debugging purposes because it will be a headache to encode every time.